README

```diff
@@ -7,11 +7,11 @@ predicates, but they are now purely graph-based and significantly extended to
 support various forms of low-level memory manipulation used in system-level
 code.  Such operations include pointer arithmetic, safe usage of invalid
 pointers, block operations with memory, reinterpretation of the memory contents,
-address alignment, etc.  The tool can be loaded into *GCC* as a *plug-in*.  This
-way you can easily analyse C code sources, using the existing build system,
-without manually preprocessing them first.  The analysis itself is, however, not
-ready for complex projects yet.  You can find the latest news on the following
-project page:
+address alignment, etc.  The tool can be loaded into *GCC* or *LLVM* as
```

Comment:
I would postpone changing this part of README until the LLVM support is reasonably stable.